app/actions/handlers.py

```diff
@@ -238,12 +238,24 @@ async def process_attachments(events, response, all_event_photos, integration):
             if response:
                 attachments_processed += len(attachments)
         except Exception as e:
-            message = f"Error while processing event attachments for event ID '{event_id['object_id']}'. Exception: {e}."
+            request = {
+                "event_id": gundi_id,
+                "attachments": attachments,
+                "integration_id": str(integration.id)
+            }
+            message = f"Error while processing event attachments for event ID '{event_id['object_id']}'. Exception: {e}. Request: {request}"
             logger.exception(message, extra={
                 "integration_id": str(integration.id),
                 "attention_needed": True
             })
-            raise e
+            await log_activity(
+                integration_id=integration.id,
+                action_id="pull_events",
+                level=LogLevel.WARNING,
+                title=message,
+                data={"message": message}
+            )
```

Comment:
Suggestion: if there's a server response, like when you get a 400, save it inside `data` too, so we can check later why we are getting the 400. Httpx saves the response on status errors, then you can get it like this: 
```suggestion
            log_data = {"message": message}
            if server_response := getattr(e, "response", None):
                log_data["server_response_body"] = server_response.text
            await log_activity(
                integration_id=integration.id,
                action_id="pull_events",
                level=LogLevel.WARNING,
                title=message,
                data=log_data
            )
```